src/util/saveArticles.ts

```diff
@@ -402,7 +422,25 @@ async function treatVideo(mw: MediaWiki, dump: Dump, srcCache: KVS<boolean>, art
     }
 
     sourceEl.setAttribute('src', newUrl);
-    return { mediaDependencies };
+
+    /* Scrape subtitle */
+    const trackEle = videoEl.querySelector('track');
+    if (!trackEle) {
```

Comment:
The construct here is too complicated, just secure `subtitles` is an empty list if no subtitle and always to to the end of the function `treatVideo`.